src/BookReader/Mode1UpLit.js

```diff
@@ -1,5 +1,5 @@
 // @ts-check
-import { customElement, html, LitElement, property, query } from 'lit-element';
+import { customElement, html, LitElement, property, query } from 'lit';
 import { styleMap } from 'lit-html/directives/style-map';
```

Comment:
It looks like these have also moved: https://lit.dev/docs/releases/upgrade/#update-directive-imports

You can try finding out where they are by searching on Google for e.g. "lit.dev styleMap". lit.dev is the new documentation site, so everything there should be for lit 2.

Reply:
☑  `styleMap` is set to be imported from `lit/directives/style-map.js`